docs/pages/404.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import Divider from '@mui/material/Divider';
+import Head from 'docs/src/modules/components/Head';
+import BrandingCssVarsProvider from 'docs/src/BrandingCssVarsProvider';
+import AppHeader from 'docs/src/layouts/AppHeader';
+import AppFooter from 'docs/src/layouts/AppFooter';
+import AppHeaderBanner from 'docs/src/components/banner/AppHeaderBanner';
+import NotFoundHero from 'docs/src/components/NotFoundHero';
+
+export default function Custom404() {
+  return (
+    <BrandingCssVarsProvider>
+      <Head
+        title="404: This page could not be found - MUI"
+        description="MUI provides a simple, customizable, and accessible library of React components. Follow your own design system, or start with Material Design."
```

Comment:
I think this should be empty, a 404 page won't be indexed in Google, so we don't want to maintain any value.

```suggestion
        description=""
```